sessionctx/variable/tidb_vars.go

```diff
@@ -1020,6 +1024,8 @@ const (
 	MaxDDLReorgBatchSize           int32  = 10240
 	MinDDLReorgBatchSize           int32  = 32
 	MinExpensiveQueryTimeThreshold uint64 = 10 // 10s
+	DefAutoBuildStatsConcurrency                   = 1
+	DefTiDBSysProcScanConcurrency                  = 1
```

Comment:
Why is one of the default value names with `TiDB`, but another one without it?

Reply:
Update the name.